tailwind.config.cjs

```diff
@@ -3,16 +3,23 @@ module.exports = {
   content: [
     "./src/pages/**/*.{js,ts,jsx,tsx,mdx}",
```

Comment:
If for whatever reason we are going to stick with the `src` directory approach, all of these lines can just be simplified to 

```javascript
"./src/**/*.{js,ts,jsx,tsx,mdx}"
```